sql/telemetry/fxa_users_last_seen_raw_v1/init.sql

```diff
@@ -11,6 +11,7 @@ SELECT
   0 AS days_seen_bits,
   0 AS days_seen_in_tier1_country_bits,
   0 AS days_registered_bits,
+  0 AS days_seen_no_monitor_bits,
```

Comment:
Similar to above, RPs other than monitor may also want to use this event, so would it make to rename this to be more general `days_seen_no_rp_enage_bits` even though its less transparent (assuming we will want to continue to filter this event for all RPs)?